src/pages/EntityDetails/Model/ApplicationsTab/ApplicationsTab.tsx

```diff
@@ -13,38 +10,19 @@ import {
   getModelApplications,
   getModelUUIDFromList,
 } from "store/juju/selectors";
-import { pluralize } from "store/juju/utils/models";
-import {
-  appsOffersTableHeaders,
-  remoteApplicationTableHeaders,
-} from "tables/tableHeaders";
 import {
   generateAppOffersRows,
   generateRemoteApplicationRows,
```

Comment:
These rows are being generated in this component as well as the table components. Is there a way the counts can be determined in this component without generating the rows?

Reply:
Fixed it!